tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -88,4 +128,23 @@ public function processed_configuration_for_array_node_1(): void
             ],
         ]);
     }
+
+    /**
+     * @test
+     */
+    public function it_overrides_default_configuration(): void
```

Comment:
This belongs in an extension test

Reply:
Not sure about how to proceed

Reply:
https://github.com/SymfonyTest/SymfonyDependencyInjectionTest#testing-a-container-extension